lib/ueberauth/strategy/google.ex

```diff
@@ -124,7 +131,7 @@ defmodule Ueberauth.Strategy.Google do
     resp = Ueberauth.Strategy.Google.OAuth.get(token, path)
 
     case resp do
-      {:ok, %OAuth2.Response{status_code: 401, body: _body}} ->
```

Comment:
@hassox Yep, it returns an :error when I tested it. I will retest in and confirm.